plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -352,6 +352,14 @@ public ContainerCreated createContainer(CreateContainerParams createContainerPar
         Set<String> exposedPorts = getExposedPorts(containerExposedPorts, imageExposedPorts);
 
         String[] envVariables = createContainerParams.getContainerConfig().getEnv();
+
+        // Replace CHE_API env var with ClusterIP for che-host
+        String cheHostClusterIp = getCheHostClusterIp();
+        for (int i = 0; i < envVariables.length; i++) {
+            if (envVariables[i].contains("CHE_API")) {
```

Comment:
personal preference, but I would probably extract this in a separate method (JUnit test for it would be also super useful here IMO)